lib/wnews.dart

```diff
@@ -1,48 +1,53 @@
+import 'dart:convert';
+
 import 'package:xpath_selector/xpath_selector.dart';
-import 'package:xpath_selector_html_parser/xpath_selector_html_parser.dart';
+import 'package:html/parser.dart' as html;
 
-/// Takes the HTML contents of something like
-/// https://en.wikipedia.org/wiki/Main_Page
-/// and extracts the items in the _In the news_ section.
+/// Parses the JSON response from
+/// https://api.wikimedia.org/feed/v1/wikipedia/en/featured/2023/10/30
+/// where date is current day and language is parameter
+/// and extracts news items.
```

Comment:
[nit] This might need a bit of stylistic improvement. Also, `date` and `language` are mentioned as wildcards, but the URL doesn't have these. You can just say "parses JSON response from URLs such as https://... as defined in [URL to wikipedia API docs]"

Reply:
Updated